packages/markdown/__tests__/index.test.js

```diff
@@ -113,29 +112,34 @@ test('glossary', () => {
   expect(shallow(markdown(`<<glossary:term>>`)).html()).toMatchSnapshot();
 });
 
-// TODO not sure if this needs to work or not?
-// Isn't it a good thing to always strip HTML?
-describe.skip('`stripHtml` option', () => {
-  test('should allow html by default', () => {
-    expect(markdown('<p>Test</p>')).toBe('<p><p>Test</p></p>\n');
-    expect(markdown('<p>Test</p>', { stripHtml: false })).toBe('<p><p>Test</p></p>\n');
-  });
+test('should escape unknown tags', () => {
+  expect(shallow(markdown('<unknown-tag>Test</unknown-tag>')).html()).toBe('<p>Test</p>');
+});
 
-  test('should escape unknown tags', () => {
-    expect(markdown('<unknown-tag>Test</unknown-tag>')).toBe(
-      '<p>&lt;unknown-tag&gt;Test&lt;/unknown-tag&gt;</p>\n',
-    );
-  });
+test('should allow certain attributes', () => {
+  expect(shallow(markdown('<p id="test">Test</p>')).html()).toBe(
+    '<p id="user-content-test">Test</p>',
+  );
+});
 
-  test('should allow certain attributes', () => {
```

Comment:
Why did you need to remove this test? Can you add back in to make sure the changes you've made are compliant with the old functionality?